hue/hue/base/hue-mysql-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: hue-mysql
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: "1Gi"
```

Comment:
I have a very little understanding about what the DB is actually used for - do you know if 1Gi is a reasonable size?

Reply:
It's used to store tables and metadata, we have been running 1 Gi in prod for some time now and it hasn't complained.

![image](https://user-images.githubusercontent.com/22432723/101179936-e8560a80-3618-11eb-8c12-f623295fe489.png)

Reply:
Same answer as at the Thrift server PR. I've used values based on the internal data hub team expertise. :slightly_smiling_face:

Hue stores mainly it's "user" data and preferences, (it's an old school Django app), query history, sample data cache for individual tables, data autocomplete and hive metadata cache, etc... In our experience this value is enough, though I don't really know how much lower we can go.